Makefile

```diff
@@ -113,7 +113,7 @@ help: ## Show this help screen
 all: manager ## Build all binaries
 
 go-licenses:
-	@which go-licenses || go install github.com/google/go-licenses@latest
+	go install github.com/google/go-licenses@latest
```

Comment:
This is going to always try to install go-licences, isn't it?

Reply:
Yeah (the same with controller-gen) but it ensures we're installing the version we want, and as far as I can tell `go install` will just exit if it's already installed

Reply:
to expand; this feels safer, and I had issues when doing this with both `controller-gen` and `go-licenses` because I had old versions installed and `which` doesn't care what version there is, just that it exists.